site/app/controllers/HomePageController.php

```diff
@@ -135,21 +136,64 @@ function(Course $course) {
      * @return Response
      */
     public function showHomepage() {
-        $courses = $this->getCourses()->json_response->json;
+        $courses = $this->getCourses()->json_response->json;        
+        $archived_courses = $courses["data"]["archived_courses"];
+        $unarchived_courses = $courses["data"]["unarchived_courses"];
+        $course_types = [$unarchived_courses, $archived_courses];
+        $rank_titles = [
+            User::GROUP_INSTRUCTOR              => "Instructor:",
+            User::GROUP_FULL_ACCESS_GRADER      => "Full Access Grader:",
+            User::GROUP_LIMITED_ACCESS_GRADER   => "Grader:",
+            User::GROUP_STUDENT                 => "Student:"
+        ];
+        $statuses = array();
```

Comment:
This and things like it can be simplified to
`$statuses = [];` which is the prefered way in PHP7